picframe/image_cache.py

```diff
@@ -9,7 +9,7 @@ class ImageCache:
 
     EXTENSIONS = ['.png','.jpg','.jpeg','.heif','.heic']
     EXIF_TO_FIELD = {'EXIF FNumber': 'f_number',
-                     'EXIF Make': 'make',
+                     'Image Make': 'make',
```

Comment:
Both tags are defined. But checking my sample images, all of them have `Image Make` defined, and most of them are not using `Exif Make`

Reply:
See [exifread](https://pypi.org/project/ExifRead/):  Note that the dictionary keys are the IFD name followed by the tag name. For example:
````
'EXIF DateTimeOriginal', 'Image Orientation', 'MakerNote FocusMode'
````

If you check [exiftool tags description](https://exiftool.org/TagNames/EXIF.html) you will find the "Group" column:
- IFD0 = Image
- ExifIFD = EXIF

Reply:
> Do you have an example image for me?

Here's a few examples. Looks like about 70% of my (frame) images have `EXIF Make` instead of `Image Make`. However, those images also have `Exif Model` instead of `Image Model`. So, with the current (mismatched) code, these have the db `make` defined but not the `model`.

Ultimately, we might have to be a bit more clever with some of the info and cascade through a number of tags looking for a definition...

![2018-10-20_145624](https://user-images.githubusercontent.com/1459347/131926003-e35d747c-61ef-4504-aa4e-e4d54afba7bd.jpg)

![2017-05-17_08-29-09](https://user-images.githubusercontent.com/1459347/131924844-fccf9478-1f79-4b51-8b52-e15e838d328d.jpg)

Reply:
Thanks for the example. Checking the [standard](https://www.cipa.jp/std/documents/e/DC-008-2012_E.pdf) I would say, that "Make" belongs to 0th IFD (Image) group. And is not defined for Exif IFD (Exif). So in my understanding your sample images are not correct. You are using Picasa, this Software is no longer maintained, isn't it?

What we can easily do is look first in the correct group and as fallback search in the other. Assuming we are focusing only on "Exif" and "Image" group. 

exifread is returning everything anyway:

````
GetImageMeta__tags:{'Image Software': (0x0131) ASCII=Picasa @ 38, 'Image ExifOffset': (0x8769) Long=46 @ 30, 'Thumbnail Compression': (0x0103) Short=JPEG ...yle) @ 558, 'Thumbnail XResolution': (0x011A) Ratio=72 @ 626, 'Thumbnail YResolution': (0x011B) Ratio=72 @ 634, 'Thumbnail ResolutionUnit': (0x0128) Short=Pixel...Inch @ 594, 'Thumbnail JPEGInterc...angeFormat': (0x0201) Long=642 @ 606, 'Thumbnail JPEGInterc...rmatLength': (0x0202) Long=5620 @ 618, 'EXIF ImageWidth': (0x0100) Short=1911 @ 56, 'EXIF ImageLength': (0x0101) Long=2875 @ 68, 'EXIF BitsPerSample': (0x0102) Signed Short=16 @ 80, 'EXIF Make': (0x010F) ASCII=NIKON...TION @ 328, 'EXIF Model': (0x0110) ASCII=NIKON D70 @ 346, 'EXIF Orientation': (0x0112) Short=Horiz...mal) @ 116, ...}
```

But I wouldn't do this for other Groups like "Thumbnail"

Reply:
> You are using Picasa, this Software is no longer maintained, isn't it?

I'm definitely not a Picasa user (and, yeah, I think it's been gone for a long time), though I did see it noted in the `Image Software` tag in one of the samples. I assume Picasa (or some part of it) must still be in use somewhere in the Google image pipeline. I must have downloaded that particular image instance from my Google Photos account instead of locating the original copy locally - that's my guess anyway...

> What we can easily do is look first in the correct group and as fallback search in the other. Assuming we are focusing only on "Exif" and "Image" group.

Agreed. That's exactly what I meant with my earlier statement...

_Ultimately, we might have to be a bit more clever with some of the info and cascade through a number of tags looking for a definition..._

> But I wouldn't do this for other Groups like "Thumbnail"

Agreed.

Reply:
> So in my understanding your sample images are not correct.

Yeah, I think I agree with you (although, the EXIF spec is a nightmare to read...) 

I'll try to get to the bottom of what's creating my questionable EXIF data, although I suspect it's coming from "ON1 Photo Raw", which is my primary RAW converter and image editor. Although, that's surprising if so, as it's a widely used package with regular updates...